### ISSUE TITLE
Predefined box positions

### ISSUE THREAD

**Issue by devourersouls** [skjolber/3d-bin-container-packing#571]
Is it possible to set certain areas that will initially be occupied by other boxes?

For example, I have a container with dimensions 1000x1000x1000.
I need to initially put two boxes with dimensions 100x100x100 in coordinates 0,0,0 and 900,0,0.

After that I need to calculate the position of other N boxes with different sizes automatically. How to implement it? I looked towards StackConstraint, but did not understand how to use it in this case.

**Comment by skjolber** [comment]
This is currently not supported, but perhaps adding some custom initializer to ExtremePoints3D would do the trick.

**Comment by devourersouls** [comment]
Ok, thanks.  Is it possible to disable the positioning of boxes at certain coordinates?  my problem is similar to issue #97

**Comment by skjolber** [comment]
@devourersouls I think the most easy approach would be to pass a list of free (possibly overlapping) areas. 

But just to be clear, you actually need to place some boxes in a rectangular container, this is not some kind of non-rectangular container shape?

**Comment by devourersouls** [comment]
@skjolber Yes, you are right, I have a non-rectangular container shape. I thought that the simplest solution would be to initially take certain places that should not be used.
